<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialClass</class>
 <widget class="QMainWindow" name="SerialClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Kontrola:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="characterControlCombo">
          <item>
           <property name="text">
            <string>E</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>N</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>O</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Kontrola przepływu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="flowCombo">
          <item>
           <property name="text">
            <string>Brak</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DTR/DSR</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RTS/CTS</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>XON/XOFF</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Prędkość (kb/s):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitrateSpin">
          <property name="minimum">
           <number>150</number>
          </property>
          <property name="maximum">
           <number>115000</number>
          </property>
          <property name="singleStep">
           <number>50</number>
          </property>
          <property name="value">
           <number>9600</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Bity stopu:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="stopBitSpin">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Pole danych (bit):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="bitSpin">
          <property name="minimum">
           <number>7</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Terminator:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="terminatorCombo">
            <item>
             <property name="text">
              <string>Brak</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CR</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>LF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>CR-LF</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Niestandardowy</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="terminatorEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maxLength">
           <number>2</number>
          </property>
          <property name="placeholderText">
           <string>Terminator</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="portsCombo"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="refreshPortsButton">
          <property name="text">
           <string>Odśwież listę portów</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Tryb transmisji:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="transmitRadio">
          <property name="text">
           <string>Nadawanie</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="receiveRadio">
          <property name="text">
           <string>Odbiór</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="openButton">
          <property name="text">
           <string>Otwórz port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="closeButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Zamknij port</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="rtdLabel">
          <property name="text">
           <string>Round Trip Delay: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pingButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Ping</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLineEdit" name="transmitTextEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="placeholderText">
           <string>Nadawanie</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="sendButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Wyślij</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="receiveTextBox">
        <property name="placeholderText">
         <string>Odbiór</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>825</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNadawanie">
    <property name="title">
     <string>Nadawanie</string>
    </property>
    <addaction name="actionTekst"/>
   </widget>
   <widget class="QMenu" name="menuOdbi_r">
    <property name="title">
     <string>Odbiór</string>
    </property>
    <addaction name="actionTekst_2"/>
    <addaction name="actionBinarny"/>
   </widget>
   <addaction name="menuNadawanie"/>
   <addaction name="menuOdbi_r"/>
  </widget>
  <action name="actionTekst">
   <property name="text">
    <string>Tekst</string>
   </property>
  </action>
  <action name="actionTekst_2">
   <property name="text">
    <string>Tekst</string>
   </property>
  </action>
  <action name="actionBinarny">
   <property name="text">
    <string>Binarny</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Serial.qrc"/>
 </resources>
 <connections/>
</ui>
